sklearn/decomposition/_dict_learning.py

```diff
@@ -1669,7 +1690,6 @@ def __init__(
         positive_dict=False,
         transform_max_iter=1000,
     ):
-
         super().__init__(
```

Comment:
Still need to revert the cosmetic line deletions. This can be done by ensuring you have `black==22.3.0` and then running `black sklearn/decomposition/_dict_learning.py`

Reply:
I double checked that I am running the correct version, before pushing and here again: 
```
(sklearn-dev) ka00ri@M1 scikit-learn % pip list | grep black
black                         22.3.0
(sklearn-dev) ka00ri@M1 scikit-learn % black sklearn/decomposition/_dict_learning.py
All done! ✨ 🍰 ✨
1 file left unchanged.
(sklearn-dev) ka00ri@M1 scikit-learn %
```

Reply:
Ah yes on second-look it is still `black` compliant. In general, even if it is `black` compliant we prefer to avoid cosmetic changes in PRs like this (i.e. empty line deletions/insertions) to better preserve `git blame` history. I've pushed a commit to revert these.

Sorry for any confusion :)